packages/plugin-ext/package.json

```diff
@@ -30,13 +31,11 @@
   "publishConfig": {
     "access": "public"
   },
-  "theiaExtensions": [
-    {
-      "backend": "lib/plugin-ext-backend-module",
-      "backendElectron": "lib/plugin-ext-backend-electron-module",
-      "frontend": "lib/plugin-ext-frontend-module"
-    }
-  ],
+  "theiaExtensions": [{
+    "backend": "lib/plugin-ext-backend-module",
+    "backendElectron": "lib/plugin-ext-backend-electron-module",
+    "frontend": "lib/plugin-ext-frontend-module"
+  }],
```

Comment:
Please revert the formatting.

Reply:
Is there a way to align our formatters? I always had this problem.